priv/perf/docker/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM elixir:1.10.4-alpine
+
+RUN apk add git curl bash maven jq \
+        autoconf \
+        automake \
+        gmp \
+        gmp-dev \
+        libtool \
+        gcc \
+        cmake \
+        gnupg \
+        alpine-sdk
+
+RUN git clone https://github.com/omgnetwork/elixir-omg
+
+WORKDIR ./elixir-omg
+
+RUN mkdir -p priv/openapitools \
```

Comment:
1. If this is the preferred way, probably consider to replace this script too: https://github.com/omgnetwork/elixir-omg/blob/master/priv/perf/scripts/generate_api_client.sh
And just run the script inside so we get consistent setup when developing perf and using this docker.

1. probably we should pin to release version tag instead of `master`?

Reply:
> If this is the preferred way, probably consider to replace this script too: https://github.com/omgnetwork/elixir-omg/blob/master/priv/perf/scripts/generate_api_client.sh
And just run the script inside so we get consistent setup when developing perf and using this docker.

I don't think it's the most convenient way because it requires installing java, maven, jq. 

2. You're right. I'll do this now